clients/ts/signalr/tests/HubConnection.test.ts

```diff
@@ -48,6 +48,46 @@ describe("HubConnection", () => {
         });
     });
 
+    describe("ping", () => {
+        it("automatically sends multiple pings", async () => {
+            const connection = new TestConnection();
+            const hubConnection = createHubConnection(connection);
+
+            hubConnection.pingIntervalInMilliseconds = 150;
+
+            try {
+                await hubConnection.start();
+                await delay(322);
+
+                const numPings = connection.sentData.filter((s) => JSON.parse(s).type === MessageType.Ping).length;
+                expect(numPings).toBeGreaterThanOrEqual(2);
+            } finally {
+                await hubConnection.stop();
+            }
+        });
+
+        it("doesn't ping while sending other messages", async () => {
```

Comment:
Let's chop this test as well, it could be subject to the same flakiness as the C# one